lib/order_helper.rb

```diff
@@ -15,6 +15,14 @@ def artworks
     @artworks ||= line_items.map(&:artwork)
   end
 
+  def artwork
+    @artwork ||= begin
+      # This is with assumption of Offer order only having one line_item. Follows
+      # similar pattern as in offer_totals.
+      line_items.first&.artwork
```

Comment:
Since there's already a method for finding the artworks, my suggestion would be to reuse it and take the first one.

```suggestion
    artworks.first
```